lighthouse-core/report/v2/report-styles.css

```diff
@@ -739,6 +745,7 @@ summary.lh-passed-audits-summary::-webkit-details-marker,
 @media screen and (max-width: 767px) {
   .lh-report {
     margin-left: 0;
+    width: 100%;
```

Comment:
sure, it seems to be a grab bag across devtools which panes shrink forever like this and which don't shrink at all and scroll as soon as it's less narrower than content, at what size do we want to stop shrinking?